moveit2.repos

```diff
@@ -3,15 +3,3 @@ repositories:
     type: git
     url: https://github.com/ros-planning/moveit_resources.git
     version: ros2
-  ros2_control:
-    type: git
-    url: https://github.com/ros-controls/ros2_control.git
-    version: 2.13.0
-  generate_parameter_library:
-    type: git
-    url: https://github.com/PickNikRobotics/generate_parameter_library.git
-    version: 0.3.0
```

Comment:
This is the currently released version, but CI is not happy with the debian (https://github.com/ros-planning/moveit2/actions/runs/3758355166/jobs/6386597488#step:12:574). @tylerjw any ideas?